src/04-modality-specific-files/06-physiological-and-other-continuous-recordings.md

```diff
@@ -97,9 +81,45 @@ sub-control01/
 {
    "SamplingFrequency": 100.0,
    "StartTime": -22.345,
-    "Columns": ["cardiac", "respiratory", "trigger"],
+   "Columns": ["cardiac", "respiratory", "trigger"],
    "cardiac": {
        "Units": "mV"
    }
 }
 ```
+
+## Recommendations for specific use cases
+
+To store pulse or breathing measurements, or the scanner trigger signal, the
+following naming conventions SHOULD be used for the column names:
+
+| Column name | Definition                                           |
+| ----------------- | ------------------------------------------------------- |
+| cardiac     | continuous pulse measurement                         |
+| respiratory | continuous breathing measurement                     |
+| trigger     | continuous measurement of the scanner trigger signal |
+
+For any other data to be specified in columns, the column names can be chosen
+as deemed appropriate by the researcher.
+
+Recordings with different sampling frequencies and/or starting times should be
+stored in separate files.
+
+If the same continuous recording has been used for all subjects (for example in
+the case where they all watched the same movie), one file MAY be used and
+placed in the root directory.
+For example, `task-movie_stim.tsv.gz`
+
+For motion parameters acquired from scanner side motion correction, the
```

Comment:
I propose to make this more explicit "from scanner side motion correction" -> "from MRI scanner side motion correction"

In other modalities other scanners are used, like PET.

Reply:
```suggestion
For motion parameters acquired from MRI scanner side motion correction, the
```